README.md

```diff
@@ -13,3 +13,23 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+Reasoning for asymptotic analysis is misleading: 
+
+- The first reason is that you could have two algorithms that have the exact asymptotic analysis. You would think these two would run at the sametime. That is not necessarily true because a lot of the time one algorithm is going to be a lot more complex than the other algorithm and it will take up a lot more time to run have a longer output time and use up a lot more resources than the first one. In actuality, these two algorithms might have the same asymptotic analysis but might not have the same output time. 
```

Comment:
Not really -- asymptotic complexity only tells you about the growth, not about the actual runtime.